extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -101,4 +105,83 @@ describe.configure().retryOnSaucelabs().run('integration amp-bind', function() {
     });
   });
 
+  describe('amp-img integration', () => {
+    it('should change src when the src attribute binding changes', () => {
+      const changeImgSrcButton =
+        fixture.doc.getElementById('changeImgSrcButton');
+      const img = fixture.doc.getElementById('image');
+      expect(img.getAttribute('src')).to.equal('https://lh3.googleusercontent' +
+        '.com/5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w300-h200-no');
```

Comment:
Is there a more pithy image URL you could use here? Does it matter than if it's valid?